game/graphic/window.hh

```diff
@@ -21,6 +21,8 @@ static inline float virtH() { return float(screenH()) / screenW(); }
 
 /// handles the window
 class Window {
+	enum class Stereo3dType {RedCyan, GreenMagenta, OverUnder};
```

Comment:
I'm not sure if it's done in other enums, but please specify the (integer) value so we're always sure which int maps to which value and we can't break it if we add a type inbetween or in front.